Jellyfin.Api/Controllers/SubtitleController.cs

```diff
@@ -113,7 +113,7 @@ public class SubtitleController : BaseJellyfinApiController
         [ProducesResponseType(StatusCodes.Status200OK)]
         public async Task<ActionResult<IEnumerable<RemoteSubtitleInfo>>> SearchRemoteSubtitles(
             [FromRoute] Guid itemId,
-            [FromRoute] string? language,
+            [FromRoute, Required] string? language,
```

Comment:
Just a mental note for everyone. These `Required` attributes may bite us in the ass since the ServiceStack API doesn't really give a shit about `IsRequired` in a validation sense.